dependencies.sh

```diff
@@ -0,0 +1,34 @@
+sudo apt-get install \
+    libatlas-base-dev gfortran \
+    libjpeg8-dev libtiff4-dev libjasper-dev libpng12-dev \
+    libavcodec-dev libavformat-dev libswscale-dev libv4l-dev \
+    build-essential cmake git pkg-config \
+    libboost-python-dev python3 python3-dev python3-pip
```

Comment:
add `cmake` to this list. At least on the debian install on the dev server, this was necessary for this script to succeed